### ISSUE TITLE
🐞[BUG] 

### ISSUE THREAD

**Issue by khakhaya** [citycoins/citycoin-ui#111]
**Describe the bug**
-> Though I am interested in mining $NYC, the action only shows an explorer about $MIA after I clicked Actions button at the https://minecitycoins.com/nyc/dashboard?chain=mainnet

**To Reproduce**
-> Steps to reproduce the behavior:
1. Go to https://minecitycoins.com/nyc/dashboard?chain=mainnet
2. Click on Wallet which was used to sign in  
3. Click the Actions button  
4. You can only see $MIA explorer while I selected $NYC. 

**Expected behavior**
-> It's supposed to show the link of 'NYC Block Explorer'

**Screenshots**
-> See attached image below. 
![image](https://user-images.githubusercontent.com/86011741/141336526-47f121f4-3a7a-40c9-94a7-3f08fac9fb2b.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 
 - Browser: Chrome  
 - Version: Doesn't matter  

 
**Additional context**

**Comment by whoabuddy** [comment]
Good catch! The profile component could use some love in general, this one should be a quick update. :pray:
